src/combat.ts

```diff
@@ -416,9 +417,16 @@ export class Macro extends LibramMacro {
   kill(): Macro {
     return (
       this.externalIf(
-        myClass() === $class`Sauceror` && have($skill`Curse of Weaksauce`),
-        Macro.trySkill($skill`Curse of Weaksauce`)
+        hippyStoneBroken() && monsterManuelAvailable(),
+        Macro.if_(
+          `monsterhpbelow ${Math.floor((100 + numericModifier("Monster Level")) / 5)}`,
```

Comment:
I think it's important to add something like
```
`(monsterid 1758 || 1759 || monsterid 1760 ) && monsterhpbelow ${Math.floor((100 + numericModifier("Monster Level")) / 5)}`,
```
because the kill() function is used against more enemies than just barf mountain native monsters.

Reply:
Fair enough, added